nb/api/normal.nb

```diff
@@ -0,0 +1,21 @@
+md`
+### normal()
+Produces a reference point along the direction normal to the shape.
+
+This can be visualized as a point.
+
+For complex shapes the normal may be somewhat arbitrary and unintuitive.
```

Comment:
😁 What is the normal of a sphere?

Reply:
For an approximate sphere it would be approximately arbitrary, I guess. :)